Frameworks/UIKit/UILabel.mm

```diff
@@ -558,11 +500,14 @@ - (UIBaselineAdjustment)baselineAdjustment {
 }
 
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes NSLineBreakByCharWrapping/NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle
+ are not supported. NSLineBreakByCharWrapping is using NSLineBreakByWordWrapping instead.
+ NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle is using NSLineBreakByTruncatingTail instead
```

Comment:
UNIMPLEMENTED_WITH_MSG has been added in ApplyLineBreakModeOnTextBlock in XamlUtilies.mm

---
In reply to: [105034771](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105034771) [](ancestors = 105034771,104821652)